<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="23508" started-at="2015-02-25T14:17:11Z" finished-at="2015-02-25T14:17:35Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="23508" started-at="2015-02-25T14:17:11Z" finished-at="2015-02-25T14:17:35Z">
      <class name="com.example.tests.PrintPhoneCompareTests">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.example.tests.PrintPhoneCompareTests@629f0666]" name="setUp" is-config="true" duration-ms="4008" started-at="2015-02-25T16:17:11Z" finished-at="2015-02-25T16:17:15Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="comparePrintphonesWithDb()[pri:0, instance:com.example.tests.PrintPhoneCompareTests@629f0666]" name="comparePrintphonesWithDb" duration-ms="18969" started-at="2015-02-25T16:17:15Z" finished-at="2015-02-25T16:17:34Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expected: <26>
     but: was <0>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expected: <26>
     but: was <0>
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.junit.Assert.assertThat(Assert.java:865)
	at org.junit.Assert.assertThat(Assert.java:832)
	at com.example.tests.PrintPhoneCompareTests.comparePrintphonesWithDb(PrintPhoneCompareTests.java:28)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- comparePrintphonesWithDb -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.example.tests.PrintPhoneCompareTests@629f0666]" name="tearDown" is-config="true" duration-ms="501" started-at="2015-02-25T16:17:34Z" finished-at="2015-02-25T16:17:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.example.tests.PrintPhoneCompareTests -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
